client/components/domain-search-v2/register-domain-step/index.jsx

```diff
@@ -596,6 +599,18 @@ class RegisterDomainStep extends Component {
 		);
 	}
 
+	updateParentClass() {
```

Comment:
We don't need this, we can use the same strategy that I used [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/steps/domains/index.jsx#L1791-L1793).